6.bulk_Morphology_Elastic_Network/0.split_data/notebooks/0.split_data_regression.ipynb

```diff
@@ -1,872 +1,912 @@
 {
```

Comment:
<p>May also consider using a dictionary to store the splits</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>